<launch>
	<arg name="camera_link" default="camera_link"/>
	<arg name="debug" default="false"/>

	<!--group ns=""-->
		<node
			pkg="robotx_vision"
			type="zed_detection"
			name="zed_detection"
			required="true"
			output="screen"
		>
		<param name="subscribed_image_topic" value="rgb/image_rect_color"/>
		<param name="subscribed_depth_topic" value="depth/image_rect_color"/>
		<param name="published_topic" value="filtered_marker_array"/>
		<param name="debug" value="$(arg debug)"/>

		</node>
	<!--/group-->
</launch>
